demo-notebooks/additional-demos/local_interactive.ipynb

```diff
@@ -50,10 +50,9 @@
    "source": [
     "# Create and submit our cluster\n",
     "# The SDK will try to find the name of your default local queue based on the annotation \"kueue.x-k8s.io/default-queue\": \"true\" unless you specify the local queue manually below\n",
-    "namespace = \"default\" # Update to your namespace\n",
     "cluster_name = \"hfgputest-1\"\n",
     "\n",
-    "cluster = Cluster(ClusterConfiguration(namespace=namespace,\n",
```

Comment:
For this demo and basic interactive the namespace variable is required later for `generate_cert.generate_tls_cert(cluster_name, namespace)`
We could still remove the namespace variable in favour of `generate_cert.generate_tls_cert(cluster_name, cluster.namespace)` WDYT?